src/prefect/infrastructure/kubernetes.py

```diff
@@ -53,7 +53,8 @@ class KubernetesJob(Infrastructure):
     """
     Runs a command as a Kubernetes Job.
 
-    Click [here](https://medium.com/the-prefect-blog/how-to-use-kubernetes-with-prefect-419b2e8b8cb2/) to see a tutorial.
+    For a guided tutorial, see [How to use Kubernetes with Prefect](https://medium.com/the-prefect-blog/how-to-use-kubernetes-with-prefect-419b2e8b8cb2/).
```

Comment:
FYI @discdiver I used a longer text sentence here because it is more accessible (sounds better when read via screen readers). The larger clickable area is also easier to navigate on mobile.